priv/repo/seeds.exs

```diff
@@ -1,11 +1,4 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     CoursePlanner.Repo.insert!(%CoursePlanner.SomeModel{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+CoursePlanner.Repo.delete_all CoursePlanner.User
```

Comment:
the main point is that you don't need a task, you can run `mix run priv/repo/seeds.exs` and `seeds.exs` is a plain elixir script

I suggest we leave it as you did before, let's keep the `seeds.exs`. I will check what to do about it